src/main/java/com/payfurl/payfurlsdk/api/CustomerApi.java

```diff
@@ -5,16 +5,7 @@
 import com.payfurl.payfurlsdk.auth.AuthHandler;
 import com.payfurl.payfurlsdk.auth.AuthType;
 import com.payfurl.payfurlsdk.http.client.HttpClient;
-import com.payfurl.payfurlsdk.models.CustomerData;
-import com.payfurl.payfurlsdk.models.CustomerList;
```

Comment:
Please do not join imports, in JB products you could switch off:
https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports 

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
Fixed.